CONTRIBUTING.md

```diff
@@ -128,45 +159,11 @@ del(src)
 ```
 ***
 
-Do not return unused values from functions. Do not use return if there's no any actions in function after it. 
```

Comment:
This is telling you to not return values arbitrarily, this is a good practice. If you do plan on using that value at a later date than it's fine. This is not a helpful thing to delete.

Reply:
Its not a sensical rule, this is like telling people to brush their teeth before bed, and not to set their house on fire.  It's not worth telling people not to do something so harmless and afaik uncommon

Reply:
That mistake appears too often actually.